subiquity/server/server.py

```diff
@@ -263,9 +270,20 @@ def __init__(self, opts, block_log_dir):
                             os.path.dirname(__file__))),
                     "examples", "snaps"),
                 self.scale_factor)
-        else:
+            self.snapd = AsyncSnapd(connection)
+        elif os.path.exists(self.snapd_socket_path):
             connection = SnapdConnection(self.root, self.snapd_socket_path)
-        self.snapd = AsyncSnapd(connection)
+            self.snapd = AsyncSnapd(connection)
+        else:
+            log.info("no snapd socket found. Snap support is disabled")
+            reload_needed = False
+            for controller in ["Refresh", "Snaplist"]:
+                if controller in self.controllers.controller_names:
+                    self.controllers.controller_names.remove(controller)
+                    reload_needed = True
+            if reload_needed:
+                self.controllers.load_all()
```

Comment:
When is this bit actually needed?

Reply:
Removed. This was a leftover from when we were still loading these 2 controllers on WSL.